package.json

```diff
@@ -17,14 +22,23 @@
   "author": "Jeff Escalante",
   "license": "MPL-2.0",
   "dependencies": {
-    "body-parser": "^1.19.0",
     "chalk": "^4.0.0",
     "chokidar": "^3.4.0",
     "commander": "^5.0.0",
     "express": "^4.17.1"
   },
+  "peerDependencies": {
+    "next": "10"
```

Comment:
I have read with more attention https://github.com/hashicorp/next-remote-watch/pull/20#discussion_r637987575 and yeah it probably make sense just to stick with a major release for each API change.

That can be reflected in this `peerDependencies`, we'll need to know in which version the API was changed to be compatible with the current implementation of `app.server.hotReloader`